automatic/awscli/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$url        = 'https://s3.amazonaws.com/aws-cli/AWSCLI32PY3-1.18.4.msi'
```

Comment:
Could you move those variables into the `$packageArgs` body to make code more alike other packages ? 

This is older variant we don't use any more.